.gitlab-ci.yml

```diff
@@ -49,6 +50,20 @@ default:
       - unknown_failure
       - api_failure
 
+
+get_agent_version:
+  stage: setup
+  image: $SETUP_IMAGE_NAME
+  script:
+    - export VERSION=$(inv -e agent.version)
```

Comment:
not sure the `-e` brings something here (I checked if it could have saved from explicitly using `print` in your task but I guess no)